content/ecosystem/_index.html

```diff
@@ -0,0 +1,444 @@
+          <div class="tags-container border-top has-border-white">
+            <span class="tags-item has-text-white has-border-white"><i class="fa-solid fa-tag"></i> SCIKIT-LEARN </span>
+            <span class="tags-item has-text-white has-border-white"><i class="fa-solid fa-tag"></i> MACHINE LEARNING</span>
+            <span class="tags-item has-text-white has-border-white"><i class="fa-solid fa-tag"></i> PYTHON / C++ </span>
+          </div>
+          <p class="has-text-white"> cuML is a library for executing machine learning algorithms on GPUs with an API that closely follows the Scikit-Learn API. Reduce the time to fit your model from hours or minutes to seconds.</p>
+          <a href="https://github.com/rapidsai/cuml">GitHub <i class="fa-solid fa-arrow-up-right"></i> </a> <br>
+          <a href="https://docs.rapids.ai/api/cuml/stable/">Documentation <i class="fa-solid fa-arrow-up-right"></i> </a>
+
+          <br><br>
+
+          <h3 class="has-text-white">libcuml</h3>
+          <p class="has-text-white">libcuml is the underlying C/C++ powering cuML.</p>
+          <a href="https://github.com/rapidsai/cuml">GitHub <i class="fa-solid fa-arrow-up-right"></i> </a> <br>
+          <a href="https://docs.rapids.ai/api/libcudf/stable/">Documentation <i class="fa-solid fa-arrow-up-right"></i> </a>
+        </div>
+        <div class="column is-one-third">
+          <h2 class="has-text-white border-bottom-white"><i class="fa-regular fa-star"></i> cuGraph</h2>
+          <div class="tags-container border-top has-border-white">
+            <span class="tags-item has-text-white has-border-white"><i class="fa-solid fa-tag"></i> NETWORKX </span>
+            <span class="tags-item has-text-white has-border-white"><i class="fa-solid fa-tag"></i> GRAPH </span>
+            <span class="tags-item has-text-white has-border-white"><i class="fa-solid fa-tag"></i> PYTHON / C++ </span>
+          </div>
+          <p class="has-text-white"> cuGraph is a library to make graph analytics available to everyone. With a python API that follows NetworkX, you can analyze networks of millions of nodes without specialized software faster than ever.</p>
+          <a href="https://github.com/rapidsai/cugraph">GitHub <i class="fa-solid fa-arrow-up-right"></i> </a> <br>
+          <a href="https://docs.rapids.ai/api/cugraph/stable/">Documentation <i class="fa-solid fa-arrow-up-right"></i> </a>
+
+          <br><br>
+
+          <h3 class="has-text-white">libcugraph</h3>
+          <p class="has-text-white">libcugraph is the underlying C/C++ powering of cuGraph.</p>
+          <a href="https://github.com/rapidsai/cugraph">GitHub <i class="fa-solid fa-arrow-up-right"></i> </a> <br>
+          <a href="https://docs.rapids.ai/api/libcugraph/stable">Documentation <i class="fa-solid fa-arrow-up-right"></i> </a>
+        </div>
+      </div>
+
+      <div id="rapids-extended"></div>
+      <h1 class="has-text-white mt-6">RAPIDS Extended</h1> 
```

Comment:
This core/extended is a bit of a weird separation, in my opinion. RAFT, for instance, is a core dependency of both cuML and cuGraph, so it's not "extending functionality." One could argue that cuSpatial "extends cuDF" but it's a pretty weak argument given that cuSpatial has a different scope of functionality. I think our product positioning should list cuSpatial as a core library as we continue to grow the library.

In my view, anything we include in the `rapids` conda metapackage is "core."

Could we get input from @beckernick or others on this question?

Reply:
There has been lots of back and forth, Im just going to remove the 'core' and 'extended' categories.